test/suite/test_txn19.py

```diff
@@ -346,12 +346,120 @@ def test_corrupt_log(self):
         newdir2 = "RESTART2"
         self.inserts([self.nrecords, self.nrecords + 1])
         expect.extend([self.nrecords, self.nrecords + 1])
-        self.copy_for_crash_restart(newdir, newdir2)
+        copy_for_crash_restart(newdir, newdir2)
         self.checks(expect)
         self.reopen_conn(newdir)
         self.checks(expect)
         self.reopen_conn(newdir2, self.conn_config)
         self.checks(expect)
 
+class test_txn19_meta(wttest.WiredTigerTestCase, suite_subprocess):
+    base_config = 'log=(archive=false,enabled,file_max=100K),' + \
+                  'transaction_sync=(enabled,method=none)'
+    conn_config = base_config
+
+    # The type of corruption to be applied
+    corruption_scen = [
```

Comment:
Would prefer to call it corruption_scenarios so we are consistent in this file. But happy to ignore.

Reply:
Fixed this, thanks!